Sources/Plasma/PubUtilLib/plDrawable/plDynaFootMgr.cpp

```diff
@@ -41,30 +41,28 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 *==LICENSE==*/
 
 #include "HeadSpin.h"
-#include "plDynaFootMgr.h"
-#include "plDynaDecal.h"
-
-#include "plCutter.h"
-
-#include "plPrintShape.h"
-
 #include "plgDispatch.h"
-
-#include "hsStream.h"
 #include "hsResMgr.h"
+#include "hsStream.h"
 #include "hsTimer.h"
 
-#include "plMessage/plDynaDecalEnableMsg.h"
+#include <array>
+
+#include "plCutter.h"
+#include "plDynaFootMgr.h"
+#include "plDynaDecal.h"
+#include "plPrintShape.h"
+
+#include "pnEncryption/plRandom.h"
 
-#include "plMessage/plAvatarFootMsg.h"
 #include "plAvatar/plArmatureMod.h"
 #include "plAvatar/plAvBrainHuman.h"
+#include "plMessage/plAvatarFootMsg.h"
+#include "plMessage/plDynaDecalEnableMsg.h"
 
-#include "pnEncryption/plRandom.h"
 static plRandom sRand;
 
-static const uint32_t kNumPrintIDs = 2;
-static const uint32_t kPrintIDs[kNumPrintIDs] =
+constexpr std::array<uint32_t, 2> kFootPrintIDs
```

Comment:
```suggestion
static constexpr uint32_t kFootPrintIDs[] =
```

(Instead of hard-coding the size)